### PR TITLE
[content update] #743-config-VM-in-GCP

### PR DESCRIPTION
**Pull request by Fernando-Iscar**
Hi all,

Following the guidelines in issue #743 , I've updated the building block concerning [Configure a VM with GPUs in Google Cloud](https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/config-vm-gcp/).

## Overview of the changes:

Since this BB is quite recent, just a few changes were applied.

- Update pics: Google Cloud Computing interface was slightly updated, hence, pics here were also updated.
- Improved readability by getting rid of some sentences, adding headlines in boxes, and rewriting some texts better.
- Added a *Learning Goals* section

### CODE REVIEW

**Review by srosh2000 — COMMENTED** [review]
Overall it looks great! Just some minor suggestions from my side

**Comment by srosh2000** [comment]
Also, I wasn't able to access this building block through the search bar. Maybe have a look if it works on your end? If not maybe some links need fixing..

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md [hunk 1]
```diff
@@ -6,26 +6,32 @@ weight: 2
...
 draft: false
-date: 2023-06-05
+date: 2023-06-05 #updated 2023-08-23
 aliases: 
   - /run/vm-on-google-cloud
   - /handle/memory-issues
   - /import/heavy-files
 ---
 
+# Learning Goals
```
[the remark this answers is not in the document]
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md:16]: Thanks for pointing it out,

✅ Re-structured overview accordingly

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md [hunk 2]
```diff
@@ -6,26 +6,32 @@ weight: 2
...
+
+- Establish and tailor a VM instance in Google Cloud with optimized configurations.
+- Use Docker and Google Colab for efficient file handling.
+- Monitor system resources and implement strategies to handle memory challenges.
+
 # Gear up your code with Google Cloud VMs
 
-Tired of encountering memory or power constraints while running your code or waiting long due to your machine's limited capacity? In this building block, you'll discover how to create and configure a simple and robust VM instance in **Google Cloud**, designed to overcome these limitations. Say goodbye to obstacles and embrace seamless computing with this comprehensive guide.
+In this building block, you will discover how to create and configure a simple and robust VM instance in [Google Cloud](https://cloud.google.com/?hl=en), designed to overcome memory or power constraints limitations. Say goodbye to obstacles and embrace seamless computing with this comprehensive guide.
```
[the remark this answers is not in the document]
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md:24]: ✅Done

File: content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md [hunk 3]
```diff
@@ -321,8 +343,7 @@ In Python, this translates to choosing dictionaries over lists when wrestling wi
 
 - **Parallelizing your Work:** Divide the task among multiple identical instances, each running a part of the code. This approach is particularly useful when your code involves training or using multiple machine learning models. For example, if you have three BERT models to run, you could use three instances.
 
-Beyond these strategies, it's always possible to leverage the scalability and flexibility of cloud services such as Google Cloud. These services allow for a dynamic allocation of resources according to your needs. 
-
+Remember that beyond these strategies, it's always possible to leverage the scalability and flexibility of cloud services such as Google Cloud. These services allow for a dynamic allocation of resources according to your needs. 
 
 ## Additional Resources
```
[the remark this answers is not in the document]
Comment (Fernando-Iscar) [reply content/building-blocks/automate-and-execute-your-work/reproducible-work/config-VM-GCP.md:348]: I see it!

✅Added a structured summary section

**Comment by Fernando-Iscar** [comment]
> I wasn't able to access this building block through the search bar

@srosh2000 Hmm strange, it does work well on my computer:

image

**Comment by Fernando-Iscar** [comment]
Hi @lachlandeer,

This building block was updated after adjusting it according to feedback. Hence, it should be ready to merge 👍

**Pull request merged**